modules/home-environment.nix

```diff
@@ -309,11 +309,11 @@ in
             relativePath="''${sourcePath#$oldGenFiles/}"
             targetPath="$HOME/$relativePath"
             if [[ -e "$newGenFiles/$relativePath" ]] ; then
-              $VERBOSE_ECHO "Checking $targetPath  exists"
+              $VERBOSE_ECHO "Checking $targetPath exists"
```

Comment:
After thinking this through, I'd prefer a colon: `Checking $targetPath: exists` :relaxed: